src/services/db/GitFileSystemService.ts

```diff
@@ -800,7 +792,7 @@ export default class GitFileSystemService {
 
   getMimeType(fileExtension: string): Result<string, MediaTypeError> {
     if (!ALLOWED_FILE_EXTENSIONS.includes(fileExtension)) {
-      err(new MediaTypeError("Unsupported file extension found"))
+      return err(new MediaTypeError("Unsupported file extension found"))
```

Comment:
previously hidden error? O.o
wah change in infering types by neverthrow seems timely

Reply:
Hmm sadly this was not because of `neverthrow` but rather due to the lack of unit tests. I noticed this when adding the unit tests for this function!